djangoProject/users/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.auth.admin import UserAdmin
+
+
+@admin.register(get_user_model())
+class CustomUserAdmin(UserAdmin):
+    list_display = ("id", "username", "first_name", "last_name", "email", "age", "is_staff")
+    pass
```

Comment:
same. `pass` is used to make function and class empty

Reply:
I know. It's just my code writing feature. I create empty function or class with 'pass' item. Then I write their functionality and don't remove 'pass' statement, if I plan to delete and rewrite something totally in future.